### PR TITLE
Enable prod cluster with throng

### PR DESCRIPTION
**Pull request by watrool**
Connected to https://github.com/rangle/rangle-starter/issues/145.

After looking through `throng` and `koa-cluster`, I am inclined to think that `pm2` is a better way for clustering in the prod mode. Unless is there any other special requirements? @SethDavenport

### CODE REVIEW

**Comment by SethDavenport** [comment]
I used pm2 for a while and had issues with it related to picking up environment variables properly.  In general I find throng to to be a lighter weight solution. I also like how throng just does clustering; daemonization is a separate concern that often gets delegated to other tools in real projects.

Can you clarify why you prefer PM2?

**Comment by watrool** [comment]
No specific reasons actually, I failed to enable throng with the starters but succeeded with PM2 ... So if PM2 is also problematic, we may focus on throng then.

**Comment by watrool** [comment]
Current solution is let all koa servers listen to same port (`process.env.PORT || 3000`), and is that desired? @SethDavenport

File: src/app.js [hunk 1]
```diff
@@ -10,31 +10,38 @@ import { rootRouter } from './routes/root.routes';
...
+  const app = setUpServer(new Koa());
+  const PORT = process.env.PORT || 3000;
+  /* istanbul ignore if */
+  if (require.main === module) {
+    winston.info(`Starting server on port ${PORT}`);
+    app.listen(PORT);
+  }
 }
+const throng = require('throng');
+throng(startFunction);
```
Comment (SethDavenport) [inline src/app.js]: I think this should probably  be

```js
function startFunction() {
  const app = setUpServer(new Koa());
  const PORT = process.env.PORT || 3000;
  winston.info(`Starting server on port ${PORT}`);
  app.listen(PORT);
}

/* istanbul ignore if */
if (require.main === module) {
  const throng = require('throng');
  throng(startFunction);
}

export const app = setUpServer(new Koa());
```

So we don't start up a bunch of dud processes in test mode.

**Comment by SethDavenport** [comment]
Yup listening to the same port should be fine.

Fix conflicts, see my comment above, and then :Shipit:

**Pull request merged**